Classes/Controllers/PBGitCommitController.m

```diff
@@ -112,19 +115,25 @@ - (IBAction)signOff:(id)sender
 {
 	NSError *error = nil;
 	GTConfiguration *config = [repository.gtRepo configurationWithError:&error];
-	NSString* userName = [config stringForKey:@"user.name"];
-	NSString* userEmail = [config stringForKey:@"user.email"];
-	if (!(userName && userEmail))
-		return [[repository windowController] showMessageSheet:@"User's name not set" infoText:@"Signing off a commit requires setting user.name and user.email in your git config"];
-	NSString *SOBline = [NSString stringWithFormat:@"Signed-off-by: %@ <%@>",
-				userName,
-				userEmail];
+	NSString *userName = [config stringForKey:@"user.name"];
+	NSString *userEmail = [config stringForKey:@"user.email"];
+	if (!(userName && userEmail)) {
+		return [[repository windowController]
+				showMessageSheet:NSLocalizedString(@"User‘s name not set",
+												   @"Title for sheet that the user’s name is not set in the git configuration")
```

Comment:
@tiennou – looks OK on my machine, I suspect a tabs vs spaces issue here.

As far as I can tell, the project uses both of those indentation styles, so it’s hard to tell what’s right and what is wrong.

My Xcode seems to default on using tabs on the project.